networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/subnets/ValidatorBasedStableSubnetSubscriber.java

```diff
@@ -95,9 +114,8 @@ public void onSlot(final UInt64 slot, final int validatorCount) {
   private Set<SubnetSubscription> adjustNumberOfSubscriptionsToNumberOfValidators(
       UInt64 currentSlot, int validatorCount) {
 
-    final int randomSubnetsPerValidator = ValidatorConstants.RANDOM_SUBNETS_PER_VALIDATOR;
     final int requiredSubnetSubscriptions =
-        min(attestationSubnetCount, randomSubnetsPerValidator * validatorCount);
+        min(attestationSubnetCount, subnetsPerNode * validatorCount);
```

Comment:
similar here, it's not validator based since this change, so we should not multiply it and we should rename the class